docs/src/content/docs/index.mdx

```diff
@@ -4,114 +4,137 @@ description: Terrateam documentation
 template: doc
 hero:
   title: Welcome to Terrateam
-  tagline: Terraform and OpenTofu GitOps CI/CD that enables teams to deliver infrastructure faster.
+  tagline: GitOps CI/CD for Terraform and OpenTofu that accelerates infrastructure delivery.
   image:
     file: ../../assets/logo.png
   actions:
     - text: Quickstart guide
       link: /quickstart-guide
       variant: primary
+    - text: Core Concepts
+      link: /getting-started/concepts
+      variant: secondary
 ---
 
 import { Steps } from '@astrojs/starlight/components';
 import { LinkCard, CardGrid, Card } from '@astrojs/starlight/components';
 
 ## What is Terrateam?
 
-Terrateam is a powerful Terraform and OpenTofu GitOps CI/CD solution that seamlessly integrates with GitHub, enabling teams to deliver infrastructure faster and more efficiently. With Terrateam, you can easily manage your infrastructure as code, collaborate with your team, and automate your deployment processes.
+Terrateam is a GitOps CI/CD platform for Terraform and OpenTofu that integrates with GitHub, enabling teams to manage infrastructure changes through pull requests.
 
 <CardGrid>
-	<Card title="Tight GitHub Integrations" icon="github">
-		Terrateam is built as a GitHub application and a GitHub Action, allowing you to manage your infrastructure directly from your GitHub repository.
+	<Card title="GitHub-Native" icon="github">
+		Built as a GitHub app and Action for seamless integration with your repos.
```

Comment:
Terrateam is built as a GitHub app and Action, ensuring seamless integration with your repositories.